src/main/java/org/skyscreamer/jsonassert/comparator/DefaultComparator.java

```diff
@@ -78,11 +81,11 @@ public void compareJSONArray(String prefix, JSONArray expected, JSONArray actual
         }
 
         if (mode.hasStrictOrder()) {
-            compareJSONArrayWithStrictOrder(prefix, expected, actual, result);
+            compareJSONArrayWithStrictOrder(prefix, expected, actual, result, this.customizations);
```

Comment:
I'm not sure whether it's the _house style_ of this codebase, but you're using `this` redundantly here (and in a few other places)